Libraries/Text/RCTShadowText.h

```diff
@@ -24,6 +24,7 @@ extern NSString *const RCTReactTagAttributeName;
 @property (nonatomic, assign) CGFloat letterSpacing;
 @property (nonatomic, assign) CGFloat lineHeight;
 @property (nonatomic, assign) NSUInteger numberOfLines;
+@property (nonatomic, copy) NSString *lineBreakMode;
```

Comment:
If you define the type as NSLineBreakMode (here and in the RCT_EXPORT_SHADOW_PROPERTY), React will automatically apply the `[RCTConvert NSLineBreakMode:]` function, so there's no need to do it manually.